wormhole-connect/src/hooks/useFetchSupportedRoutes.ts

```diff
@@ -1,42 +1,45 @@
-import { useEffect } from 'react';
-import { useSelector, useDispatch } from 'react-redux';
-import { useDebounce } from 'use-debounce';
-
-import { RouteState, setRoutes } from 'store/transferInput';
+import { useEffect, useMemo, useState } from 'react';
+import { useSelector } from 'react-redux';
 
 import type { RootState } from 'store';
 import config from 'config';
 import { getTokenDetails } from 'telemetry';
 
-const useFetchSupportedRoutes = (): void => {
-  const dispatch = useDispatch();
+type HookReturn = {
+  supportedRoutes: string[];
+  isFetching: boolean;
+};
+
+const useFetchSupportedRoutes = (): HookReturn => {
+  const [routes, setRoutes] = useState<string[]>([]);
+  const [isFetching, setIsFetching] = useState<boolean>(false);
 
   const { token, destToken, fromChain, toChain, amount } = useSelector(
     (state: RootState) => state.transferInput,
   );
 
   const { toNativeToken } = useSelector((state: RootState) => state.relay);
 
-  const [debouncedAmount] = useDebounce(amount, 500);
```

Comment:
this debounce is redundant with another one in `AmountInput`. causes unneeded delay.